GuessTheNumber.java

```diff
@@ -20,5 +21,104 @@
  * "There are X players with better scores") or if he's at the top of the table.
  * Before the next game, the top 3 scores (which are greater than 0) will be printed
  */
+import java.util.Scanner;
+
 public class GuessTheNumber {
+	public static void main(String[] args) {
+		Scanner input = new Scanner(System.in);
+
+		System.out.print("Please enter number of digits for the game (2 to 10): ");
+		int targetNumberSize = input.nextInt();
+		while (targetNumberSize < 2 || targetNumberSize > 10) {
+			System.out.println("You can only enter value from 2 to 10. Please try again.");
+			targetNumberSize = input.nextInt();
+		}
+
+		int[] userGuessArray = new int[targetNumberSize];
+		int[] targetNumber = new int[targetNumberSize];
+		for (int i = 0; i < targetNumberSize; i++) {
+			targetNumber[i] = 10;
+		}
+
+		// Generate target number
+		for (int i = 0; i < targetNumberSize; i++) {
+			int digitToAdd = (int) (Math.random() * 10);

```

Comment:
👍 for googling something else to generate random numbers
👎 for the Math.random choice :) - You should use Random.nextInt(10)

Reply:
Changed to Random